### PR TITLE
Add level requirements to certain modules

### PR DESCRIPTION
**Pull request by alazymeme**
Added level 1000 requirements to most moderation modules, upped some more critical ones to 1500.

Fixes https://github.com/pajbot/pajbot/issues/492

Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable
- [ ] Documentation in docs/ or install-docs/ was updated, if applicable

### CODE REVIEW

**Comment by alazymeme** [comment]
@DatGuy1 Are there anymore changes you wanted?

File: pajbot/modules/pnsl.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ class PNSLModule(BaseModule):
     NAME = "Run P&SL lists"
     DESCRIPTION = "Run P&SL lists through the !runpnsl command"
     CATEGORY = "Moderation"
+    CONFIGURE_LEVEL = 1500
```
Comment (Felanbird) [inline pajbot/modules/pnsl.py:18]: I personally think this module should be 1000, people who utilize this module are most likely 1000 or 2000, and almost never 1500.

**Comment by RAnders00** [comment]
Level 500 is "Moderator". 1000 is Broadcaster/Super Moderator I think.

So denying moderators access to moderation modules is wrong.

This change also breaks everyone's setup where they have their actual moderators as level 500.

I think a better solution would be to allow a different (lower) level to also have access to edit modules, but only the "fun" modules.

**Comment by alazymeme** [comment]
> Level 500 is "Moderator". 1000 is Broadcaster/Super Moderator I think.
> 
> So denying moderators access to moderation modules is wrong.
> 
> This change also breaks everyone's setup where they have their actual moderators as level 500.
> 
> I think a better solution would be to allow a different (lower) level to also have access to edit modules, but only the "fun" modules.
> 
> I appreciate the effort though.

I completely disagree. This would be the same system as streamelements. Mods can enable/disable modules, but cannot modify unless they have direct access to the streamer's dashboard.

**Pull request closed without merging**